### ISSUE TITLE
Cannot open debug version with mac + safari

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/ratio-and-proportion#508]
**Test device**
MacBook Air m1 chip

**Operating System**
macOS 12.5.1

**Browser**
Safari 15.6.1

**Problem description**
For https://github.com/phetsims/qa/issues/831 but also seen in other recent tests (ex. https://github.com/phetsims/qa/issues/830), the debug version doesn't open with safari. An error message appears in the console. It also happens after I build a version on master. 

<img width="499" alt="Screen Shot 2022-09-06 at 1 44 42 PM" src="https://user-images.githubusercontent.com/87318828/188727623-5f60a0a1-2ec8-4999-a4b7-7f18fccadddd.png">

This was reproduced by @samreid as well on 9/1. From slack:

>[Sam Reid](https://app.slack.com/team/U6DKKJ2FN)
  [5 days ago](https://phetsims.slack.com/archives/C6HPE0J91/p1662047317705099?thread_ts=1662038671.383559&cid=C6HPE0J91)
https://phet-dev.colorado.edu/html/gravity-and-orbits/1.6.0-dev.5/phet/gravity-and-orbits_all_phet_debug.html launched twice for me on Mac/Chrome with no problems.  
[@Nancy Salpepi](https://phetsims.slack.com/team/U027AF93XE1)
 what URL did you see the problem with?

>[Sam Reid](https://app.slack.com/team/U6DKKJ2FN)
  [5 days ago](https://phetsims.slack.com/archives/C6HPE0J91/p1662047394510879?thread_ts=1662038671.383559&cid=C6HPE0J91)
In Mac/Safari, I get this error:

![image](https://user-images.githubusercontent.com/87318828/188727392-8aa6fbc5-4893-4a8f-8200-4f702ca02c3a.png)

>[Sam Reid](https://app.slack.com/team/U6DKKJ2FN)
  [5 days ago](https://phetsims.slack.com/archives/C6HPE0J91/p1662047416436599?thread_ts=1662038671.383559&cid=C6HPE0J91)
Does the debug version contain code that is incompatible with Safari?

>[Nancy Salpepi](https://app.slack.com/team/U027AF93XE1)
  [5 days ago](https://phetsims.slack.com/archives/C6HPE0J91/p1662047431241739?thread_ts=1662038671.383559&cid=C6HPE0J91)
I have no issue with chrome either. The issue is with safari specifically.

>[Sam Reid](https://app.slack.com/team/U6DKKJ2FN)
  [5 days ago](https://phetsims.slack.com/archives/C6HPE0J91/p1662047433115779?thread_ts=1662038671.383559&cid=C6HPE0J91)
And do we ever give the debug versions to clients, or are they for our team? (edited)

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Ratio and Proportion‬
URL: https://phet-dev.colorado.edu/html/ratio-and-proportion/1.2.0-dev.31/phet/ratio-and-proportion_all_phet.html
Version: 1.2.0-dev.31 2022-08-24 16:50:40 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.6.1 Safari/605.1.15
Language: en-US
Window: 1322x704
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by samreid** [comment]
@zepumph and I built a local version with `grunt --brands=phet --minify.minify=false` and observed this stack trace when running:

http://localhost/main/ratio-and-proportion/build/phet/ratio-and-proportion_all_phet_debug.html

<img width="789" alt="image" src="https://user-images.githubusercontent.com/679486/188989641-96b90c0e-64d1-4904-aceb-2c9571ac8548.png">

We also observed that for the local and published versions, the sim launched with no problems when the JS console was open, and failed with that error when the JS console was closed.  Keyword: heisenbug

**Comment by samreid** [comment]
We are in uncharted waters.  The google search `"cannot access uninitialized variable" "webpack" safari` has only 4 results.  We know it is a webpack problem because thing work in unbuilt mode with `public static` class attributes.  Running transpile even for the debug versions "patches" the problem though (or makes it so webpack is OK with it), but does make the debug code a bit uglier (combines statements onto single lines).

**Comment by zepumph** [comment]
I also want to add that we updated to webpack 5.74 and the problem still persisted.

**Comment by jonathanolson** [comment]
Dev meeting: we'll add minimal transpilation, AND notify webpack.

**Comment by zepumph** [comment]
@samreid will you please pull all and confirm this is fixed to unblock this issue?

**Comment by samreid** [comment]
I pulled, inspected the commit, built with `grunt --brands=phet` and tested locally at http://localhost/main/ratio-and-proportion/build/phet/ratio-and-proportion_all_phet_debug.html in Safari with "Empty Caches" and no dev tools open. It launched and ran well.  If that all sounds good, perhaps this issue can be closed.

**Comment by zepumph** [comment]
Yes that seems right to me, if I felt like I was <1 hour to a simple, reproducible case to give to webpack, then I think it would be worth it, but this workaround seems more than reasonable given our current priorities and schedule. Closing

**Comment by samreid** [comment]
Noting that Safari 16 was released today.  I tested https://phet-dev.colorado.edu/html/ratio-and-proportion/1.2.0-dev.31/phet/ratio-and-proportion_all_phet_debug.html with dev tools closed and cache empty, and it worked OK on safari Version 16.0 (17614.1.25.9.10, 17614).

@zepumph should we change anything?

**Comment by zepumph** [comment]
If that is the case, I think we should revert the commit. I did so above. Can you please pull and build locally and confirm things still have the behavior you expect?
